README.md

```diff
@@ -113,11 +113,10 @@ This runs several multi-process tests:
  - A [test](integration/map_integration_test.go) that starts a Trillian server
    in Map mode, sets various key:value pairs and checks they can be retrieved.
  - A [test](integration/log_integration_test.go) that starts a Trillian server
-   in Log mode, together with a signer, logs many leaves, and checks they are
-   integrated correctly.
+   in Log mode, together with a signer, logs many leaves, and checks they are integrated correctly.
  - A [test](integration/ct_integration_test.go) that starts a set of Trillian
    servers in Log mode, plus a signer and a set of Certificate Transparency
-   personality servers, then runs tests that exercise all of the
+   personality servers then run tests that exercise all of the
```

Comment:
I think it was fine before - you've removed a comma separating the dependent clause, and the conjugation for a _[singluar]_ test should be "runs"...